packages/otelbin/src/app/layout.tsx

```diff
@@ -18,6 +18,7 @@ const inter = Inter({ subsets: ["latin"] });
 export const metadata: Metadata = {
 	title: "OTelBin – by Dash0",
 	description: "Edit, visualize and share OpenTelemetry Collector configurations",
+	metadataBase: new URL("https://otelbin.io/"),
```

Comment:
**Issue:** While this is the the correct value for our production env, it will not be the correct value for others that are deploying OTelBin or for our preview envs.

Do we need this setting, or can we just remove it?

Reply:
You're correct; we can remove it. In other fields, such as images, the full URL is used.

Reply:
Upon further examination, the next server encounters an error without the presence of 'metadataBase.' This functionality is implemented in `/social-preview/[id]/page.tsx`, where metadata is utilized with the url.origin